pkg/model/manifest.go

```diff
@@ -1057,3 +1059,15 @@ func (m *Manifest) IsDeployDefault() bool {
 	}
 	return false
 }
+
+// SetManifestDefaultsFromDev sets context and namespace from the dev
+func (m *Manifest) SetManifestDefaultsFromDev() {
+	if len(m.Dev) == 1 {
+		for _, devInfo := range m.Dev {
+			m.Context = devInfo.Context
+			m.Namespace = devInfo.Namespace
```

Comment:
What happens if `m.Context` or `m.Namespace` are set?

Reply:
I am testing it on my machine. It looks like `okteto up -n namespace-different-than-the-one-from-manifest` panics

Reply:
It's weird because `okteto up -f okteto.yml -n namespace-different-than-the-one-from-manifest` does work 🤔